lib/jobs/obm-control.js

```diff
@@ -38,17 +38,17 @@ function obmControlJobFactory(
      * @param {String} taskId
      * @constructor
      */
-    var ObmControlJob = function ObmControlJob(options, context, taskId) {
+   var ObmControlJob = function ObmControlJob(options, context, taskId) {
```

Comment:
total nit, but slight indent error here. Not that javascript cares, just looks funny